src/manifests/component_manifest.py

```diff
@@ -51,19 +51,24 @@ def __to_dict__(self) -> List[Dict[Any, Any]]:
         as_dict: Callable[[ComponentType], dict] = lambda component: component.__to_dict__()
         return list(map(as_dict, self.values()))
 
-    def select(self, focus: str = None) -> Iterator[ComponentType]:
+    def select(self, focus: List[str] = None) -> Iterator[ComponentType]:
         """
         Select components.
 
-        :param str focus: Choose one component.
+        :param List[str] focus: Choose some components.
         :return: Collection of components.
         :raises ValueError: Invalid platform or component name specified.
         """
+        if focus and len(focus) > 0:
+            invalid = [item for item in focus if item not in self]
+            if len(invalid) > 0:
+                raise ValueError(f"Unknown component{'s'[:len(invalid) != 1]}={','.join(invalid)}.")
```

Comment:
Using `len(invalid)^1` will return 0 or 1.
Using `len(invalid) != 1` will return True or False.

Apparently `'s'[:True]` will resolve as `'s'[:1]` and print `s`.
Great thanks.

Reply:
```
In Python 3.x True and False are keywords and will always be equal to 1 and 0.
```

Not in Python 2.x tho glad our code base is in 3.